types/feathers-rethinkdb/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "@types/rethinkdbdash": "github:types/rethinkdbdash"
```

Comment:
I don't think we want to add dependencies on github repositories. Those types should be ported here (with the author's permission) or created fresh.

Reply:
OK, in that case I will close this in favor of using [types/rethinkdbdash](https://github.com/types/rethinkdbdash) and [types/rethinkdb](https://github.com/types/rethinkdb) stored outside of DefinitelyTyped